astronomer/providers/amazon/aws/example_dags/example_redshift_sql.py

```diff
@@ -54,17 +57,24 @@ def delete_redshift_cluster() -> None:
             SkipFinalClusterSnapshot=True,
         )
 
-        while True:
-            if get_cluster_status() == "deleting":
-                time.sleep(30)
-                continue
+        while get_cluster_status() == "deleting":
+            time.sleep(30)
+
     except ClientError as exception:
-        logging.exception("Error deleting redshift cluster")
-        raise exception
+        if exception.response.get("Error", {}).get("Code", "") == "ClusterNotFound":
+            logging.error(
+                "Cluster might have already been deleted. Error message is: %s",
+                exception.response["Error"]["Message"],
+            )
+        else:
+            logging.exception("Error deleting redshift cluster")
+            raise
```

Comment:
what about raising `raise exception` this might have more info?

Reply:
`logging.exception` will contain that details

Reply:
in this case, `raise` will have the same impact as `raise exception`; it's just a short hand for that.

<img width="699" alt="Screenshot 2022-05-16 at 12 50 25 PM" src="https://user-images.githubusercontent.com/10206082/168539664-5a90bd84-ddd5-4b5a-a2d3-7c13147d2d80.png">